HypeSocial/HypeSocial/Feature/Home/RootViewController.swift

```diff
@@ -17,13 +16,12 @@ final class RootViewController: UIViewController, NVActivityIndicatorViewable {
 
     override func viewDidAppear(_ animated: Bool) {
         super.viewDidAppear(animated)
-        view.backgroundColor = .blue
-        let activityData = ActivityData(message: "Get Info from Outlook", type: .ballScaleRippleMultiple)
-        NVActivityIndicatorPresenter.sharedInstance.startAnimating(activityData, nil)
+        view.backgroundColor = .levioGreen
+        showLoading(withMessage: Strings.Root.getInfoFromOutlook)
         tempServiceMSAL.start()
 
         DispatchQueue.main.asyncAfter(deadline: DispatchTime.now() + 2) {
```

Comment:
Hey I really know you are think in let this here, but please make it right! Remove the DispatchQueue.